src/test/resources/GET.feature

```diff
@@ -1,15 +1,27 @@
 @Get
 
-Feature: Get users information
+Feature: Get operation
 
   Background:
     Given I am working on "QA" environment
     And I am targeting "at-sce-api" service
+    And I have access to "at-sce-db" database
 
 
-@US_001 @1
-  Scenario: Get all users
-  Given I want to retrieve all users
-  When I send a GET request
-  Then the status code should be "200"
-  And information retrieved from service should match with DB
```

Comment:
Why did you delete this scenario? Is it no longer used or needed?

Reply:
That was an example of how we are supposed to write an escenario, I can put it back if you want

Reply:
Got it. Then is ok you have removed it from the file.